### PR TITLE
fixed export of views

### PR DESCRIPTION
**Pull request by JanTvrdik**
backported from clevis fork, views must be exported after all tables

From [documentation](https://dev.mysql.com/doc/refman/5.7/en/create-view.html):

> Any table or view referred to in the definition must exist.

### CODE REVIEW

File: src/MySQLDump.php [hunk 1]
```diff
@@ -73,15 +73,19 @@ public function write($handle = NULL)
...
+			if ($row[1] === 'VIEW') {
+				$views[] = $row[0];
+			} else {
+				$tables[] = $row[0];
+			}
 		}
 		$res->close();
 
-		$this->connection->query('LOCK TABLES `' . implode('` READ, `', $tables) . '` READ');
+		$tables = array_merge($tables, $views); // views must be last
```
Comment (dg) [inline src/MySQLDump.php]: Why did you remove the lock?

Comment (JanTvrdik) [inline src/MySQLDump.php]: fixed, mistake in when resolving conflict

**Comment by dg** [comment]
Thanks
